terraform/applications/etherpad/variables.tfvars

```diff
@@ -0,0 +1,6 @@
+cost_center   = "1440"
```

Comment:
I suggest deleting either this file or the defaults in variables.tf – having the same values in both places seems redundant.

Reply:
sure removed tfvars then.